File: C/PRAK110-2410817210029-BADJRADAKAHERDINANSYAHRAHARDJO.c
#include <stdio.h>
#include <math.h>

int main () {
    int alas, tinggi;
    alas = 5;
    tinggi = 12;
    int miring = sqrt((alas * alas) + (tinggi * tinggi));
    int keliling = alas + tinggi + miring;
    int luas = (alas * tinggi) / 2;

    printf("Diketahui :\n"); 
    printf("Alas = %d cm\n", alas); 
    printf("Tinggi = %d cm\n", tinggi); 
    printf("\n");
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", tinggi); 
    printf("Sisi B = %d cm\n", miring); 
    printf("Sisi C = %d cm\n", alas); 
    printf("Keliling = %d cm\n", keliling); 
    printf("Luas = %d cm\n", luas);

return 0;
}

File: Python/PRAK110-2410817210029-BADJRADAKAHERDINANSYAHRAHARDJO.py
import math
alas = 5
tinggi = 12
miring = math.sqrt(alas**2 + tinggi**2)
keliling = (alas + tinggi + miring)
luas = ((alas * tinggi) / 2)
print("Diketahui :")
print("Alas = %d cm" % (alas))
print("Tinggi = %d cm" % (tinggi))
print()
print("Jawab :")
print("Sisi A = %d cm" % (tinggi))
print("Sisi B = %d cm" % (miring))
print("Sisi C = %d cm" % (alas))
print("Keliling = %d cm" % (keliling))
print("Luas = %d cm" % (luas))
